src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -356,6 +356,9 @@ Whether Harvesting (OAI) service is enabled
         If 'DEPENEDENT' then the DOI will be the Dataset DOI plus a file DOI with a slash in between.
         */
         DataFilePIDFormat, 
+        /* Json array of supported languages
+        */
+        Languages,
```

Comment:
This new `:Languages` setting needs to be documented. This is where we document them: http://guides.dataverse.org/en/4.9.1/installation/config.html#database-settings